enterprise/websvc.rest/src/org/netbeans/modules/websvc/rest/codegen/resources/DefaultResource.template

```diff
@@ -1,4 +1,23 @@
-<#-- FreeMarker template (see http://freemarker.org/) -->
+<#-- FreeMarker template (see http://freemarker.org/)
```

Comment:
Please let the license "comment" stay clear of other text. Many problems regarding license cleanup come from slightly different texts. If you think, that the "FreeMarker template (see http://freemarker.org/)" is important, please move it to a second comment, separate from the license text.